RecoTracker/TkNavigation/test/NavigationSchoolAnalyzer.cc

```diff
@@ -20,42 +20,39 @@
 #include <memory>
 
 // user include files
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-
+#include "DataFormats/ForwardDetId/interface/BTLDetId.h"
+#include "DataFormats/ForwardDetId/interface/ETLDetId.h"
+#include "DataFormats/MuonDetId/interface/CSCDetId.h"
+#include "DataFormats/MuonDetId/interface/DTChamberId.h"
+#include "DataFormats/MuonDetId/interface/RPCDetId.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
Not needed?

Reply:
removed in [8915076](https://github.com/cms-sw/cmssw/pull/36980/commits/891507613510060934ecf314218f15b296bf7bfc)